dawn/renderer/utils/utils.js

```diff
@@ -60,3 +62,31 @@ export const timings = {
   TELEOP: 120,
   SEC: 1000,
 };
+
+let _log = null;
+class Logger {
+  constructor() {
+    const d = new Date();
+    this.log_file = fs.createWriteStream(`./${d.valueOf()}-dawn.log`, { flags: 'w' });
+    this.log_file.write(d.toString());
+    this.lastStr = '';
+  }
+
+  log(output) {
+    console.log(output);
+    output = String(output);
+    if (output !== this.lastStr) {
+      this.log_file.write(`\n${output}`);
+      this.lastStr = output;
+    } else {
+      this.log_file.write('*');
+    }
+  }
+}
+
+export const logging = () => {
```

Comment:
Per my comment from earlier: This is a function, so if you wanted to access the logger you'd probably need `logging().log` with your current implementation; it'd probably make more sense to just set `logging = new Logger();` and remove `_log`.

Also, could you possibly remove the duplicate version of `Logger` from `Ansible`, export this version, and give this one a constructor argument for file name?